tools/snp.sh

```diff
@@ -88,13 +89,25 @@ IMAGE="${IMAGE:-${LAUNCH_WORKING_DIR}/${GUEST_NAME}.img}"
 GENERATED_INITRD_BIN="${SETUP_WORKING_DIR}/initrd.img"
 
 # URLs and repos
-AMDSEV_URL="https://github.com/ryansavino/AMDSEV.git"
-AMDSEV_DEFAULT_BRANCH="snp-latest-fixes"
-AMDSEV_NON_UPM_BRANCH="snp-non-upm"
+[ -e /etc/os-release ] && . /etc/os-release
+
+if [ "$ID" = "debian" ] || [ "$ID_LIKE" = "debian" ]; then
+  AMDSEV_URL="https://github.com/ryansavino/AMDSEV.git"
```

Comment:
What is currently missing from Mike's AMDSEV that we cannot use his? Can we open PRs to get any required changes in there?

Reply:
Please review the following commits I made for the rhel and fedora fix snp-latest branch:

https://github.com/LakshmiSaiHarika/AMDSEV/commits/rhel-fedora-fix

Please let me know if my commits are fine so that I can go ahead with  PR request for AMDSEV

Reply:
These look okay. Please make sure that this encompasses all changes needed so we don't have to make another PR to AMDSEV. We should have a discussion about non-upm.